docs/users_guide/ocean/test_groups/global_ocean.rst

```diff
@@ -476,7 +476,7 @@ The mesh has 12-km resolution around Antarctica and 30-km resolution elsewhere.
 The mesh includes the :ref:`global_ocean_ice_shelf_cavities` around Antarctica
 in the ocean domain.
 
-.. image:: images/sowisc12to30.png
+.. image:: images/sowisc12to60.png
```

Comment:
Unrelated to this PR but an error in the docs build that should be fixed.  (We would ideally plot the update mesh.)